### PR TITLE
error console debugging

### LINKED ISSUE #60: error console bugs

### LINKED ISSUE #64: fixing synchronization of TM and GTM
when the user scrolls down or up, we clearly can see there are not synching

### PR DESCRIPTION
**Pull request by hastizolfagharei**
2 bugs fixed

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3993,6 +4001,9 @@ def GTM_CellChange_fun(self):
         if value == "":
             if (row in Data["General"]["DataSection"][str(column)]):
                 Data["General"]["DataSection"][str(column)].pop(row)
+                item.setBackground(Qt.red)
+                list_item = self.add_delete_error_in_TM(key, mode = "add")     
+                Data["error_in_TM"][key] = list_item
```
Comment (farzad1132) [inline main.py:4021]: **Why** we need these lines of codes and **what** they do?
It seems to changes cell color to red at GTM every time a cell gets cleared, its right?

**Review by farzad1132 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (hastizolfagharei) [reply main.py:4021]: because we shouldn't have demands without source or destination...

**Comment by hastizolfagharei** [comment]
gtm and tm tables sliderbars were not exactly syncing with each other. it's fixed now.

**Comment by hastizolfagharei** [comment]
save changes button color didn't work properly. fixed

**Comment by hastizolfagharei** [comment]
whenever i click on  any card in listwidget it sets excel amounts not what i entered before.
because:
int(row) vs str(row) in data

**Review by farzad1132 — APPROVED** [review]
well done

**Pull request merged**